fast/stages/02-networking-nva/data/nva-startup-script-ew1.tftpl

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+echo 'Enabling IP forwarding'
+sed '/net.ipv4.ip_forward=1/s/^#//g' -i /etc/sysctl.conf &&
+sysctl -p /etc/sysctl.conf &&
+/etc/init.d/procps restart
+
+echo 'Setting Routes'
+ip route add ${landing-untrusted-default-ew4-cidr} via ${landing-untrusted-default-ew1-gw} dev ens4
```

Comment:
Can't you use a single template, and change the vars you pass to `templatefile()` to get two versions?